source/systems/gaea_user_guide.rst

```diff
@@ -2197,7 +2197,7 @@ servers (MDS), and can lead to a MDS locking up affecting all files
 served on that MDS.
 
 Another method used for sharing files is referencing files stored in
-pdata (*/lustre/f2/pdata*) directly.  Users should copy files out of
+pdata (*/lustre/F5/pdata*) directly.  Users should copy files out of
```

Comment:
@underwoo - what's the equivalent on F5 for this?  (I don't see any `/gpfs/f5/pdata`)

Reply:
Pdata no longer exists. It is up to the projects to place their shared data in either their world or project shared directories.

Reply:
Thank you Seth. Removed the paragraph.